phantomgames/Search/SearchGameViewController.swift

```diff
@@ -6,20 +6,23 @@
 //
 
 import UIKit
+import SDWebImage
 
 class SearchGameViewController: UIViewController {
-    
+   
     // MARK: IBOutlets
     
     @IBOutlet weak private var tableView: UITableView!
-    private lazy var viewModel = SearchGameViewModel(repository: SearchGameRepository(),
-                                                     delegate: self)
+    private lazy var viewModel = SearchGameViewModel(repository: SearchGameRepository(), delegate: self)
+    // MARK: UI Component
+    private let searchController = UISearchController(searchResultsController: nil)
```

Comment:
what is this UISearchController  for?

Reply:
it enables the search bar

Reply:
do you not have to link it from the storyboard?

Reply:
a search bar is not a ui kit component so I can't link it via storyboard